client/components/domains/contact-details-form-fields/index.jsx

```diff
@@ -450,25 +453,27 @@ export class ContactDetailsFormFields extends Component {
 
 				<div className="contact-details-form-fields__extra-fields">{ this.props.children }</div>
 
-				<FormFooter>
-					<FormButton
-						className="contact-details-form-fields__submit-button"
-						disabled={ ! countryCode || disableSubmitButton }
-						onClick={ this.handleSubmitButtonClick }
-					>
-						{ labelTexts.submitButton || translate( 'Submit' ) }
-					</FormButton>
-					{ onCancel && (
+				{ ! this.props.hideFooter && (
+					<FormFooter>
 						<FormButton
-							className="contact-details-form-fields__cancel-button"
-							type="button"
-							isPrimary={ false }
-							onClick={ onCancel }
+							className="contact-details-form-fields__submit-button"
+							disabled={ ! countryCode || disableSubmitButton }
+							onClick={ this.handleSubmitButtonClick }
 						>
-							{ translate( 'Cancel' ) }
+							{ labelTexts.submitButton || translate( 'Submit' ) }
 						</FormButton>
-					) }
-				</FormFooter>
+						{ onCancel && (
+							<FormButton
+								className="contact-details-form-fields__cancel-button"
+								type="button"
+								isPrimary={ false }
+								onClick={ onCancel }
+							>
+								{ translate( 'Cancel' ) }
+							</FormButton>
+						) }
+					</FormFooter>
+				) }
```

Comment:
This component is also used in the edit domain contact details form. Just mentioning it to make sure that we keep it in mind in case there are any regressions. 

I checked it manually and it the visuals/form features work ok.

Reply:
Yeah, I'm aware of it. I added only one optional prop and it will have no effect when the prop is not set.